lisa/util/__init__.py

```diff
@@ -51,6 +52,20 @@ class UnsupportedOperationException(Exception):
     ...
 
 
+class UnsupportedDistroException(Exception):
+    """
+    This exception is used to indicate that a test case does not support the testing
+    distro.
+    """
+
+    def __init__(self, os_name: str, os_version: VersionInfo) -> None:
```

Comment:
`name` and `version` are enough. It's also inconsistent between `distro` and `os`.